packages/worker/src/api/routes/global/users.js

```diff
@@ -51,20 +51,20 @@ function buildInviteAcceptValidation() {
 router
   .post(
     "/api/global/users",
-    adminOnly,
+    builderOrAdmin,
     users.buildUserSaveValidation(),
     controller.save
   )
   .post(
     "/api/global/users/bulk",
-    adminOnly,
+    builderOrAdmin,
```

Comment:
It's not ideal for us to allow developers to effectively manage all user endpoints around creation and deletion. It's more of an unfortunate consequence of us not using the correct rest semantics, where we could allow builder on a `PUT` request only. 

I think the correct thing to do here is to 
- allow the `builderOrAdmin` on the two `POST` requests
- remove from the `DELETE`
- add request validation at the controller layer to check:
   - is this a create - i.e. no `_id` specified
   - is the `ctx.user` not an admin
   - if so, throw a manual error, to mimic the `builderOrAdmin` middleware behaviour

Reply:
It would also be good to add some RBAC tests for each of the changed endpoints to ensure the desired user type can access the endpoint, and will be rejected if not. i.e.

```ts
describe("POST /api/global/users, () => {
   it("rejects non-admin on create", () => { })
   it("allows builder or admin on edit", () => { })
})

describe("POST /api/global/users/bulk, () => {
   it("rejects non-admin create", () => { })
   it("allows builder or admin on edit", () => { })
})
```